### PR TITLE
Add 4.14 workflow for CNV Interop testing

### PR DESCRIPTION
**Pull request by rabin-io**

### CODE REVIEW

**Comment by rabin-io** [comment]
/pj-rehearse periodic-ci-RedHatQE-interop-testing-cnv-4.14-ocp-cnv-odf-4.14-cnv-odf-tests-aws-upi-ocp413

**Comment by rabin-io** [comment]
/pj-rehearse periodic-ci-RedHatQE-interop-testing-cnv-4.14-ocp-cnv-odf-4.14-cnv-odf-tests-aws-upi-ocp413

**Comment by rabin-io** [comment]
/pj-rehearse periodic-ci-RedHatQE-interop-testing-cnv-4.14-ocp-cnv-odf-4.14-cnv-odf-tests-aws-upi-ocp413

**Comment by rabin-io** [comment]
/pj-rehearse periodic-ci-RedHatQE-interop-testing-cnv-4.14-ocp-cnv-odf-4.14-cnv-odf-tests-aws-upi-ocp414

**Comment by rabin-io** [comment]
/pj-rehearse periodic-ci-RedHatQE-interop-testing-cnv-4.14-ocp-cnv-odf-4.14-cnv-odf-tests-aws-upi-ocp414

File: ci-operator/config/RedHatQE/interop-testing/RedHatQE-interop-testing-cnv-4.14__ocp-cnv-odf-4.14.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      memory: 200Mi
+tests:
+- as: cnv-odf-tests-aws-upi-ocp414
+  cron: 4 14 * * SAT
+  steps:
+    cluster_profile: aws-interop-qe
+    env:
+      BASE_DOMAIN: release-ci.cnv-qe.rhood.us
+      OCP_VERSION: "4.14"
+      ODF_OPERATOR_CHANNEL: stable-4.13
```
Comment (myakove) [inline ci-operator/config/RedHatQE/interop-testing/RedHatQE-interop-testing-cnv-4.14__ocp-cnv-odf-4.14.yaml:42]: ODF version is 4.13, is this ok?

Comment (rabin-io) [reply ci-operator/config/RedHatQE/interop-testing/RedHatQE-interop-testing-cnv-4.14__ocp-cnv-odf-4.14.yaml:42]: For 4.14 yes, for 4.15 I need to change it to use a ICSP to pull from the dev channel

**Comment by rabin-io** [comment]
/pj-rehearse periodic-ci-RedHatQE-interop-testing-cnv-4.14-ocp-cnv-odf-4.14-cnv-odf-tests-aws-upi-ocp414

**Comment by rabin-io** [comment]
/pj-rehearse periodic-ci-RedHatQE-interop-testing-cnv-4.14-ocp-cnv-odf-4.14-cnv-odf-tests-aws-upi-ocp414

**Comment by rabin-io** [comment]
/pj-rehearse periodic-ci-RedHatQE-interop-testing-cnv-4.14-ocp-cnv-odf-4.14-cnv-odf-tests-aws-upi-ocp414

**Comment by rabin-io** [comment]
/pj-rehearse periodic-ci-RedHatQE-interop-testing-cnv-4.14-ocp-cnv-odf-4.14-cnv-odf-tests-aws-upi-ocp414

**Comment by rabin-io** [comment]
/pj-rehearse periodic-ci-RedHatQE-interop-testing-cnv-4.15-ocp-cnv-odf-4.15-cnv-odf-tests-aws-upi-ocp415

**Comment by rabin-io** [comment]
/pj-rehearse periodic-ci-RedHatQE-interop-testing-cnv-4.15-ocp-cnv-odf-4.15-cnv-odf-tests-aws-upi-ocp415

**Review by myakove — APPROVED** [review]

**Pull request merged**